repository-structure.md

```diff
@@ -0,0 +1,75 @@
+# Common Repository Structure
+
+Hyperledger projects are required to maintain a standard set of files in each repository. This
+document describes the required and recommended files.
+
+## Required Files with Specified Content
+
+Repositories MUST have these files with the specific content in the linked files, or a file with a
+link to the specified content with minimal exposition. These files MUST be at the root of the
+repository.
+
+-   [`LICENSE`](https://www.apache.org/licenses/LICENSE-2.0.txt) - https://www.apache.org/licenses/LICENSE-2.0.txt\
+    (Unless an exception has been made by the Hyperledger Governing Board)
+-   [`CODE_OF_CONDUCT.md`](https://wiki.hyperledger.org/community/hyperledger-project-code-of-conduct) - https://wiki.hyperledger.org/community/hyperledger-project-code-of-conduct
+-   [`SECURITY.md`](https://wiki.hyperledger.org/display/SEC/Defect+Response) - https://wiki.hyperledger.org/display/SEC/Defect+Response
+
+## Required Files with Variable Content
+
+Repositories MUST have these files. Named files MUST be at the root of the repository, and may have
+format suffixes such as `.md`, `.rst`, or `.txt`.
+
+-   `README` \
+    A description of the project and contain information or links to information such as
+    -   A reference to the Apache license (required).
+    -   The current and important past releases
+    -   Documentation for developers and users
+-   `MAINTAINERS` \
+    A list of all current maintainers with contact info. [A separate document](MAINTAINERS-guidelines.md)
+    covers the specifics.
+-   `CONTRIBUTING` \
+    Directions on how to contribute code to the project, or a pointer to the Wiki page with that information.
+-   `CHANGELOG` \
+    A human readable list of recent changes. Changes should at least include the current release. This
+    file may be maintainer curated or mechanically produced.
+-   Continuous Integration / Continuous Delivery (CICD) configurations \
+    Configurations needed to run CICD on Hyperledger provided systems.
```

Comment:
Do we still have Hyperledger provided CICD systems?

Reply:
Github actions is paid for by hyperledger, as is CircleCI. So the intent (allowing for future changes) is to say run it as github actions or on circlci. That way if jenkins came back (it won't), or drone.io or some other CI provider was paid for the TSC wouldn't have to amend this document.

Is there different verbiage that should be used?

Reply:
To be clear, GitHub actions are free; right now, Hyperledger pays for CircleCI and Azure Pipelines.

This is a huge issue, as several projects run in opaque Jenkins silos where inspection of build errors is not possible without an account.

Reply:
Good to know. I knew that things were being pushed towards GitHub actions. I did not realize there were still CircleCI and Azure Pipelines being used. I was commenting on the "Hyperledger provided systems", but it seems this is still valid.